ansibullbot/wrappers/defaultwrapper.py

```diff
@@ -94,6 +94,7 @@ def __init__(self, github=None, repo=None, issue=None, cachedir=None, file_index
         self._assignees = False
         self._comments = False
         self._committer_emails = False
+        self._committer_logins = False
```

Comment:
Putting both bool and list into the same var is probably not the best idea. How about
```suggestion
        self._committer_logins = set()
```

Reply:
It repeats the same pattern as all the other class vars, but I guess it could be switched.

Reply:
I'll file a bug to fix that across the board later.